homeassistant/components/shelly/light.py

```diff
@@ -48,21 +67,42 @@ def brightness(self):
         return int(brightness / 100 * 255)
 
     @property
-    def supported_features(self):
-        """Supported features."""
-        return self._supported_features
+    def color_temp(self) -> Optional[float]:
+        """Return the CT color value in mireds."""
+        if self.control_result:
+            color_temp = self.control_result["colorTemp"]
+        else:
+            color_temp = self.block.colorTemp
+        return int(color_temperature_kelvin_to_mired(color_temp))
+
+    @property
+    def min_mireds(self) -> float:
+        """Return the coldest color_temp that this light supports."""
+        return color_temperature_kelvin_to_mired(2700)
+
+    @property
+    def max_mireds(self) -> float:
+        """Return the warmest color_temp that this light supports."""
+        return color_temperature_kelvin_to_mired(6500)
 
-    async def async_turn_on(
-        self, brightness=None, **kwargs
-    ):  # pylint: disable=arguments-differ
+    async def async_turn_on(self, **kwargs) -> None:
         """Turn on light."""
         params = {"turn": "on"}
-        if brightness is not None:
-            params["brightness"] = int(brightness / 255 * 100)
+        if ATTR_BRIGHTNESS in kwargs:
+            tmp_brightness = kwargs.get(ATTR_BRIGHTNESS)
```

Comment:
Since you know it exists, us square brackets
```suggestion
            tmp_brightness = kwargs[ATTR_BRIGHTNESS]
```

Reply:
I've built this based on https://github.com/home-assistant/core/blob/dev/homeassistant/components/hive/light.py#L107.
Good to know. Will use square brackets in the future